app/components/notification-list/component.js

```diff
@@ -1,7 +1,24 @@
 import Component from '@ember/component';
+import { inject as service } from '@ember/service';
 import { serviceLinks } from 'ember-osf/const/service-links';
+import { task } from 'ember-concurrency';
 
 export default Component.extend({
     serviceLinks,
+    store: service(),
     classNames: ['content'],
+
+    init() {
```

Comment:
It makes testing a little easier if you use `didReceiveAttrs` instead of `init`.